uw-frame-components/portal/messages/partials/announcement-mascot.html

```diff
@@ -45,30 +45,30 @@
       </a>
     </md-menu-item>
     <div ng-show="vm.announcements.length > 0">
-      <md-menu-item ng-repeat="announcement in vm.announcements | orderBy:['-goLiveDate'] | limitTo:3"
-                    class="top-bar-menu-item">
-        <div class="menu-item-content" layout="row" layout-align="space-between center">
-          <div layout="column" layout-align="center start" flex="80">
-            <span>{{ announcement.titleShort }}</span>
-            <span class="menu-item-description">{{ announcement.descriptionShort }}</span>
-            <div ng-if="announcement.moreInfoButton || announcement.actionButton" class="compact-buttons">
-              <md-button class="md-raised md-accent"
-                         ng-if="announcement.actionButton"
-                         ng-click="vm.takeButtonAction(announcement.actionButton.url)"
-                         ng-disabled="announcement.actionButton.disabled"
-                         target="_blank"
-                         rel="noopener noreferrer">
-                {{ announcement.actionButton.label }}
-              </md-button>
-              <md-button class="md-default"
-                         ng-if="announcement.moreInfoButton"
-                         ng-href="{{ announcement.moreInfoButton.url }}"
-                         target="_blank"
-                         rel="noopener noreferrer">
-                {{ announcement.moreInfoButton.label }}
-              </md-button>
-            </div>
+    <md-menu-item ng-repeat="announcement in vm.announcements | orderBy:['-goLiveDate'] | limitTo:3"
+                  class="top-bar-menu-item">
+      <div class="menu-item-content" layout="row" layout-align="space-between center">
+        <div layout="column" layout-align="center start" flex="80">
+          <span>{{ announcement.titleShort }}</span>
+          <span class="menu-item-description">{{ announcement.descriptionShort }}</span>
+          <div ng-if="announcement.moreInfoButton || announcement.actionButton" class="compact-buttons">
+            <md-button class="md-raised md-accent"
+                       ng-if="announcement.actionButton"
+                       ng-disabled="announcement.actionButton.disabled"
+                       ng-click="vm.takeButtonAction(announcement.actionButton)"
+                       target="_blank"
+                       rel="noopener noreferrer">
+              {{ announcement.actionButton.label }}
```

Comment:
a11y: insufficient context problem: in isolation this button label provides insufficient context to understand what it does. Add *what* to home? If a screen reader is jumping between clickables, it's gonna read this label, and they won't have enough context to evaluate whether to actuate it.